Code/Framework/AzCore/AzCore/UnitTest/Mocks/MockITime.h

```diff
@@ -88,6 +90,16 @@ namespace AZ
             return AZ::Time::ZeroTimeUs;
         }
 
+        virtual void SetElapsedTimeMs([[maybe_unused]] TimeMs timeMs) override
+        {
+            return;
```

Comment:
could remove virtual and keep override. that would probably require touching all the other methods though, ha

Reply:
Yeah, I'll leave as is.